js/Model.js

```diff
@@ -412,6 +412,7 @@ define(
 				this.resolvingConceptSetExpression(true);
 				const resolvingPromise = httpService.doPost(sharedState.vocabularyUrl() + 'resolveConceptSetExpression', expression)
 					.then(callback)
+					.then(this.resolvingConceptSetExpression(false))
```

Comment:
@anthonysena, this is a function call which will be resolved before `httpService.doPost` and `callback` complete

Reply:
So, are you saying the proper syntax would be something like:

```
.then(() => this.resolvingConceptSetExpression(false))
```

Or do some sort of `bind` where the function will be invoked with specific parameters?  Is there a standard approach we should take for this sort of coding pattern?

Reply:
Right, either option should work. Honestly, don't have a preference here

Reply:
Thanks @pavgra and @chrisknoll for the feedback here. My focus initially was simply setting the `resolvingConceptSetExpression` to false at the time of the vocabulary service call. I've revised this to use the suggested approach:

.then(() => this.resolvingConceptSetExpression(false))

And all is well. PR is updated. Thanks!